README.md

```diff
@@ -303,6 +312,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Notes**:  See this page for their [FAQ's](https://substrate.dev/hacktoberfest/faq/) or this page for their [official announcement](https://www.parity.io/hacktoberfest-2019-parity/).
 
 
+### 4 Merged PRs
+
+#### **Devfolio** (limited edition t-shirts and stickers)
+
+- **Requirements**: Get atleast four pull request merged into any project [react-otp-input](https://github.com/devfolioco/react-otp-input), [devfolio-gatsby-starter](https://github.com/devfolioco/devfolio-gatsby-starter) or [react-intl-tel-input](https://github.com/patw0929/react-intl-tel-input) to win stickers and for four exceptional pull requests get a chance to earn their LIMITED EDITION t-shirts!
```

Comment:
Their blog post lists [4 repos](https://devfolio.co/blog/hacktoberfest-2019-devfolio/) and please double check your spelling here. :wink: